test/test_autograd.py

```diff
@@ -121,11 +121,8 @@ def forward(ctx, tensor1, pyscalar, tensor2):
             @staticmethod
             @once_differentiable
             def backward(ctx, grad_output):
+                self.assertFalse(torch.is_grad_enabled())
```

Comment:
I thought this would be true in the forward only?  Or is there not really a difference between a forward and once_differentiable backward?